app/subscriber/src/components/layout/constants/MenuItemNames.ts

```diff
@@ -0,0 +1,11 @@
+export enum MenuItemNames {
+  /** The name of the menu item for the home page. */
+  Home = 'Home',
```

Comment:
Do we need this still if we're using an array of items?

Reply:
This will be a maintenance/regression issue.  Change the `SidebarMenuItems` from an array into an object.  Then each menu will be a property name `activeItem === menuItems.topStories.label`.  In the `SidebarMenuItems` you can either use the `keysof` to get the menu items as an array or you can add a function `toArray()` that does the work for you in the object.